pxtblocks/plugins/renderer/constants.ts

```diff
@@ -19,14 +19,14 @@ export class ConstantProvider extends Blockly.zelos.ConstantProvider {
     /**
      * Color of the glow outline around highlighted blocks.
      */
-    HIGHLIGHT_GLOW_COLOUR = '#FFF200';
+    HIGHLIGHT_GLOW_COLOUR = '#ffa200';
```

Comment:
We could switch colour based on experiment.

We need to discuss the selection colour and theme hackery soon.